index.html

```diff
@@ -1142,8 +1142,8 @@ <h2><dfn>Populating and maintaining the coalesced and predicted events lists</df
         <h1><dfn data-lt="compatibility mouse events">Compatibility mapping with mouse events</dfn></h1>
         <p>The vast majority of web content existing today codes only to Mouse Events. The following describes an algorithm for how the user agent MAY map generic pointer input to mouse events for compatibility with this content.</p>
         <p>The compatibility mapping with mouse events are an OPTIONAL feature of this specification. User agents are encouraged to support the feature for best compatibility with existing legacy content. User agents that do not support compatibility mouse events are still encouraged to support the <code>click</code> and <code>contextmenu</code> events (see the note below).</p>
+        <p>The <code>click</code> event, defined in [[UIEVENTS]], and the <code>contextmenu</code> event are not considered <a>compatibility mouse events</a> as they are typically tied to user interface activation and are fired from other input devices, like keyboards.</p>
```

Comment:
We have just added `contextmenu` event to UIEvents, so we can now clean this up a bit:

> The `click` and <code>contextmenu</code> events, defined in [[UIEVENTS]], are not...